crispy_forms/templates/bootstrap4/layout/checkboxselectmultiple.html

```diff
@@ -5,7 +5,7 @@
 
     {% for choice in field.field.choices %}
     <div class="{%if use_custom_control%}custom-control custom-checkbox{% if inline_class %} custom-control-inline{% endif %}{% else %}form-check{% if inline_class %} form-check-inline{% endif %}{% endif %}">
-        <input type="checkbox" class="{%if use_custom_control%}custom-control-input{% else %}form-check-input{% endif %}{%if is_bound %} is-{% if field.errors %}in{%endif%}valid{% endif %}"{% if choice.0 in field.value or choice.0|stringformat:"s" in field.value or choice.0|stringformat:"s" == field.value|default_if_none:""|stringformat:"s" %} checked="checked"{% endif %} name="{{ field.html_name }}" id="id_{{ field.html_name }}_{{ forloop.counter }}" value="{{ choice.0|unlocalize }}" {{ field.field.widget.attrs|flatatt }}>
```

Comment:
We should keep the failing `is-invalid` style. So how about:
**old:** {%if is_bound %} is-{% if field.errors %}in{%endif%}valid{% endif %}
**new:** {% if field.errors %}is-invalid{% endif %}
Make sense?

Reply:
Yeah, that makes sense. Re-added `is-invalid` styling.